src/org/ods/Context.groovy

```diff
@@ -49,6 +49,9 @@ interface Context {
     // Whether to send notifications if the build is not successful.
     boolean getNotifyNotGreen()
 
+    // Enable/disable notifications
+    def setNotifyNotGreen(boolean notifyNotGreen)
```

Comment:
Better use a specific types in a public interface instead of `def`.

Reply:
Reused the existing style. Anyhow, fixed it for the whole class.